forage-android/src/main/java/com/joinforage/forage/android/network/data/CheckBalanceRepository.kt

```diff
@@ -74,27 +75,24 @@ internal class CheckBalanceRepository(
         var attempt = 1
 
         while (true) {
-            logger.debug("Polling check balance message status. Attempt: $attempt.")
+            logger.i("Polling for balance check response for Payment Method $paymentMethodRef")
```

Comment:
include `contentId` in log? Also, worth including `paymentMethodRef` in log tag / Context for easier DataDog search?